### PR TITLE
PSD 849 Add Pod Identity resources

### PR DESCRIPTION
**Pull request by FayToward**
https://eaflood.atlassian.net/browse/PSD-849

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -397,6 +397,33 @@ secret:
...
 ```
+### Azure Identity template
+* Template file: `_azure-identity.yaml`
+* Template name: `ffc-helm-library.azure-identity`
+
+#### Required values
+
+The following values need to be set in the parent chart's `values.yaml` in addition to the globally required values [listed above](#all-template-required-values):
+
+```
```
Comment (paulsimonandrews) [inline README.md:408]: This should be:
```
azureIdentity:
  name:
  resourceID: 
  clientID: 
```

File: README.md [hunk 2]
```diff
@@ -397,6 +397,33 @@ secret:
...
+
+### Azure Identity Binding template
+* Template file: `_azure-identity-binding.yaml`
+* Template name: `ffc-helm-library.azure-identity-binding`
+
+#### Required values
+
+The following values need to be set in the parent chart's `values.yaml` in addition to the globally required values [listed above](#all-template-required-values):
+
+```
```
Comment (paulsimonandrews) [inline README.md:424]: This should be:
```
azureIdentityBinding:
  name: 
  selector: 
```

File: README.md (hunk 1 above)
Comment (FayToward) [reply README.md:408]: Thank you - changes made

File: README.md (hunk 2 above)
Comment (FayToward) [reply README.md:424]: Thank you - changes made

File: ffc-helm-library/templates/_azure-identity-binding.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{{- define "ffc-helm-library.azure-identity-binding.tpl" -}}
+{{- $requiredMsg := include "ffc-helm-library.default-check-required-msg" . -}}
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentityBinding
+metadata:
+  name: {{ required (printf $requiredMsg "azureIdentityBinding.name") .Values.azureIdentityBinding.name | quote }}
+  namespace: {{ required (printf $requiredMsg "namespace") .Values.namespace | quote }}
```
Comment (johnwatson484) [inline ffc-helm-library/templates/_azure-identity-binding.yaml:7]: metadata should also include labels.

```
labels:
    {{- include "ffc-helm-library.labels" . | nindent 4 }}
```

File: ffc-helm-library/templates/_azure-identity.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{{- define "ffc-helm-library.azure-identity.tpl" -}}
+{{- $requiredMsg := include "ffc-helm-library.default-check-required-msg" . -}}
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: {{ required (printf $requiredMsg "azureIdentity.name") .Values.azureIdentity.name | quote }}
+  namespace: {{ required (printf $requiredMsg "namespace") .Values.namespace | quote }}
```
Comment (johnwatson484) [inline ffc-helm-library/templates/_azure-identity.yaml:7]: Labels should also be attached to this resource

**Review by johnwatson484 — CHANGES_REQUESTED** [review]
Looks good, just needs the standard labels adding to each resource.

File: ffc-helm-library/templates/_azure-identity-binding.yaml (hunk 3 above)
Comment (FayToward) [reply ffc-helm-library/templates/_azure-identity-binding.yaml:7]: added

File: ffc-helm-library/templates/_azure-identity.yaml (hunk 4 above)
Comment (FayToward) [reply ffc-helm-library/templates/_azure-identity.yaml:7]: added

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**